helm_deploy/hmpps-interventions-service/templates/_envs.tpl

```diff
@@ -7,6 +7,9 @@ env:
   - name: SERVER_PORT
     value: "{{ .Values.image.port }}"
 
+  - name: SPRING_PROFILES_ACTIVE
+    value: "{{ .Values.env.SPRING_PROFILES_ACTIVE }}"
```

Comment:
I think this will break if this isn't defined in `helm_deploy/values-dev.yaml`

Reply:
argh